src/convertToObject.js

```diff
@@ -12,7 +12,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+
+  const properties = sourceString
+    .split(';')
+    .map(property => (
+      property
+        .trimLeft()
+        .split(':')
+        .map(elementOfProperty => elementOfProperty.trim())
+    ));
+
+  for (let i = 0; i < properties.length; i++) {
+    if (properties[i].length > 1) {
+      styles[properties[i][0]] = properties[i][1];
+    }
+  }
```

Comment:
Це можна зробити і без циклу
```suggestion
  styles = Object.fromEntries(properties);
```

Reply:
Артур непоганий спосіб підказав